Bio/Align/emboss.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2008-2016 by Peter Cock.  All rights reserved.
+#
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+"""Bio.AlignIO support for "emboss" alignment output from EMBOSS tools.
```

Comment:
This is clearly based on ``Bio/AlignIO/EmbossIO.py`` but this docstring needs tweaking slightly now, both here and a few lines down (although that hypothetical new high level API does not exist yet).